src/quest_manager/tests/test_models.py

```diff
@@ -66,13 +66,25 @@ def test_xp_sum(self):
         # create some quests as part of the test campaign
         baker.make(Quest, campaign=self.category, xp=1)  # included in sum
         baker.make(Quest, campaign=self.category, xp=2)  # included in sum
-        baker.make(Quest, xp=4)  # NOT included in sum because n a different campaign
-        baker.make(Quest, visible_to_students=False, xp=8)  # NOT included in sum becuase not visible
-        baker.make(Quest, archived=True, xp=16)  # NOT included in sum becase archived
+        baker.make(Quest, xp=4)  # NOT included in sum because in a different campaign
+        baker.make(Quest, visible_to_students=False, xp=8)  # NOT included in sum because not visible
+        baker.make(Quest, archived=True, xp=16)  # NOT included in sum because archived
 
         # check that the XP sum is correct
         self.assertEqual(self.category.xp_sum(), 3)
 
+    def test_quest_count(self):
+        """ Test that the number of all quests in a campaign is returned correctly """
+
```

Comment:
Can you add another assert at the top to verify the quest count is zero before you add any? (if that's the state at the start, I'm not sure)

Reply:
```suggestion
    def test_quest_count(self):
        """ Test that the number of all quests in a campaign is returned correctly """

        # assert that the current campaign has no quests
        self.assertEqual(self.category.quest_count(), 0)
```
assertion added